pkg/crc/machine/config/config.go

```diff
@@ -7,15 +7,17 @@ type MachineConfig struct {
 	BundleName string
 
 	// Virtual machine configuration
-	Name            string
-	Memory          int
-	CPUs            int
-	DiskSize        int
-	ImageSourcePath string
-	ImageFormat     string
-	SSHKeyPath      string
-	KubeConfig      string
-	SharedDirs      []string
+	Name              string
+	Memory            int
+	CPUs              int
+	DiskSize          int
+	ImageSourcePath   string
+	ImageFormat       string
+	SSHKeyPath        string
+	KubeConfig        string
+	SharedDirs        []string
+	SharedDirPassword string
```

Comment:
so when we deal with sshfs, you are only able to use a single password for each of them?

Reply:
similar to https://github.com/code-ready/crc/pull/3308#discussion_r957988343 i was not thinking about `sshfs` internally that much, was only thinking about the user facing config options as of now, when implementing `sshfs` we might have to change this